lib/collections/collections.js

```diff
@@ -425,7 +425,7 @@ var settings = new SimpleSchema ({
     radarrQUALITYPROFILEID: {
         type: Number,
         label: "Download quality profile for Radarr",
-        defaultValue: 1,
+        defaultValue: '1',
```

Comment:
Reason for changing to a string versus just a number, as the type is defined as a number, the schema expects a number--could cause issues on new installations.